tests/test_matprep_compare.py

```diff
@@ -291,3 +349,48 @@ def test_all_bads(self, pyprep_noisy, matprep_noisy):
         pyprep_bads_all = sorted(pyprep_noisy.get_bads())
         matprep_bads_all = sorted(matprep_noisy["bads"]["all"])
         assert pyprep_bads_all == matprep_bads_all
+
+
+class TestCompareRobustReference(object):
+    """Compare the results of Reference to the equivalent MatPREP code.
+
+    These comparisons use input data that's already had adaptive line noise
+    removal done to the signal, so any differences in results will be due to
+    differences in the robust referencing code itself.
+
+    """
+
+    # TODO: once final interpolation is separated out in PyPREP, add test just
+    # for interpolation code
```

Comment:
Can you open an issue for that once this PR is merged? That'd make it easier to track, as code comments are easily forgotten

Reply:
I've updated the first comment in #73 to mention this, does that work? https://github.com/sappelhoff/pyprep/issues/73

Reply:
yes thank you - that's good enough!